src/components/Farmhand/Farmhand.js

```diff
@@ -271,19 +270,21 @@ const applyPriceEvents = (valueAdjustments, priceCrashes, priceSurges) => {
  */
 
 export default class Farmhand extends Component {
-  // Bind event handlers
-
-  localforage = localforage.createInstance({
-    name: 'farmhand',
-    description: 'Persisted game data for Farmhand',
-  })
-
   /*!
    * @member farmhand.Farmhand#state
    * @type {farmhand.state}
    */
   state = this.createInitialState()
 
+  static defaultProps = {
+    localforage: localforage.createInstance({
+      name: 'farmhand',
+      description: 'Persisted game data for Farmhand',
+    }),
```

Comment:
This change effectively lets us dependency-inject LocalForage, as can be seen in some of the new tests in this PR. There are a few benefits to this:

  - We can drop the `initialState` construct, which is a feature that was designed specifically for testing and therefore not ideal.
  - We can hydrate the app with testing state by leveraging standard features of the game (loading a save file).